src/vcpkg/vcpkgpaths.cpp

```diff
@@ -267,9 +257,10 @@ namespace vcpkg
                 }
                 else
                 {
-                    Checks::exit_with_message(VCPKG_LINE_INFO,
-                                              "Error: Invalid bundle definition.\n%s\n",
-                                              maybe_bundle_doc.error()->to_string());
+                    msg::println_error(msg::format(msgInvalidBundleDefinition)
+                                           .append_raw('\n')
+                                           .append_raw(maybe_bundle_doc.error()->to_string()));
+                    Checks::exit_fail(VCPKG_LINE_INFO);
```

Comment:
```suggestion
                    Checks::msg_exit_with_error(VCPKG_LINE_INFO, msg::format(msgInvalidBundleDefinition)
                                           .append_raw('\n')
                                           .append_raw(maybe_bundle_doc.error()->to_string()));
```

? I'm not sure on this one.